intermediate/challenge-1/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk
```

Comment:
why is there no explicit tag ?

Reply:
FROM adoptopenjdk/openjdk11:alpine-jre
is fine.